### ISSUE TITLE
Wondering if there's a safe way to shutdown the ESP services?

### ISSUE THREAD

**Issue by ychen2u** [intel/Edge-Software-Provisioner#5]
Hi, Maintainers,

How can we shutdown ESP services safely after I finished the node provisioning?

There's no script to tear down the services.

I tried to run "docker-compose down", but after the service containers are down, I failed to delete the ESP folder and got the following "Device busy" issue:
```
rm: cannot remove 'esp/template/nginx/index.html': Device or resource busy
rm: cannot remove 'esp/data/srv/tftp': Device or resource busy
rm: cannot remove 'esp/data/usr/share/nginx/html/tftp': Device or resource busy
```

I have to manually "sudo umount" those locations, then I can clean up the folders.

Please help to check this issue and let me know if anything wrong.

Thanks!

**Comment by ychen2u** [comment]
The tested version is 1.6.2.

**Comment by dvintel** [comment]
Have you tried running "run.sh -d"?

https://github.com/intel/Edge-Software-Provisioner/blob/b0704ad6cfa19156897907d52cd5001b140f83aa/run.sh#L26

**Comment by ychen2u** [comment]
Thanks, will try!

**Comment by inteltiger** [comment]
the following worked for me
# umount /root/dek/esp/data/usr/share/nginx/html/tftp
# rm -rf /root/dek/esp/data/usr/share/nginx/html/tftp

**Comment by ychen2u** [comment]
Yep, this works. But I think the script should take care of these mounts.

**Comment by zhajianl** [comment]
@ychen2u 
Patch for this issue:
https://github.com/intel/Edge-Software-Provisioner/pull/30
